CHANGELOG.md

```diff
@@ -718,7 +718,7 @@ where each class name or style property can be managed separately.
     - If using `pytest`, the decorator allows you to parametrize Django or Components settings.
     - The decorator also serves as a stand-in for Django's `@override_settings`.
 
-    See the API reference for [`@djc_test`](https://django-components.github.io/django-components/0.131/reference/testing_api/#djc_test) for more details.
+    See the API reference for [`@djc_test`](https://django-components.github.io/django-components/0.131/reference/testing_api/#django_components.testing.djc_test) for more details.
```

Comment:
It might be hard to see, so unless mentioned otherwise, these changes are just URL updates. Like here the fragment is changed from `#djc_test` to `#django_components.testing.djc_test`.

Sidenote - It would be nicer to have the fragment here just `#djc_test`, but this  would need to be configured in mkdocs somehow, and I didn't bother looking into that.